packages/react-core/src/components/CodeBlock/examples/CodeBlock.md

```diff
@@ -5,210 +5,14 @@ cssPrefix: pf-c-code-block
 propComponents: ['CodeBlock', 'CodeBlockAction', 'CodeBlockCode']
 ---
 
-import CopyIcon from '@patternfly/react-icons/dist/esm/icons/copy-icon';
-import PlayIcon from '@patternfly/react-icons/dist/esm/icons/play-icon';
```

Comment:
I think the build is failing because you will need to add these back. It's a weird quirk, but the icons needed in any of the examples need to be additionally imported at the beginning of our example files.

Reply:
Thank you @nicolethoen  - that is a weird quirk! I've added these imports back in as suggested 👍